src/activities/assignments/AssignmentEntity.js

```diff
@@ -258,4 +258,28 @@ export class AssignmentEntity extends Entity {
 		];
 		await performSirenAction(this._token, action, fields);
 	}
+
+	annotationToolsAvailable() {
+		return this._entity.getSubEntityByClasses(['annotations', 'enabled']);
```

Comment:
Applies elsewhere too, but there is a `Classes` in `hypermedia-constants` that these could be added to.